src/git-duet/git_config.go

```diff
@@ -114,6 +122,33 @@ func (gc *GitConfig) RotateAuthor() (err error) {
 	return nil
 }
 
+func makeRotatedTeam(teamAuthor *Pair, teamCommitter *Pair) (newAuthor *Pair, newCommitter *Pair) {
+	authorInitials := teamAuthor.Initials
+	authorName := teamAuthor.Name
+	authorEmail := teamAuthor.Email
+	committerInitialsList := strings.Split(teamCommitter.Initials, ", ")
+	committerNamesList := strings.Split(teamCommitter.Name, ", ")
+	committerEmailsList := strings.Split(teamCommitter.Email, ", ")
+	InitialsList := append(committerInitialsList, authorInitials)
+	EmailsList := append(committerEmailsList, authorEmail)
+	NamesList := append(committerNamesList, authorName)
+	newAuthorInitials, newTeamCommitterInitials := InitialsList[0], strings.Join(InitialsList[1:], ", ")
+	newAuthorName, newTeamCommitterName := NamesList[0], strings.Join(NamesList[1:], ", ")
+	newAuthorEmail, newTeamCommitterEmail := EmailsList[0], strings.Join(EmailsList[1:], ", ")
+	newAuthor = &Pair{
+		Name:     newAuthorName,
+		Email:    newAuthorEmail,
+		Username: "Hello World",
+		Initials: newAuthorInitials}
```

Comment:
Please maintain a coding style that is consistent with the rest of the code base.  The ending `}` should appear on its own line